extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -142,6 +145,32 @@ describes.realWin('consent-ui', {
     });
   });
 
+  describe('placeholder', () => {
+    it('should add a placeholder element' +
+      ' while loading CMP Iframe', () => {
+
+      const config = dict({
+        'promptUISrc': 'https//promptUISrc',
+      });
+      consentUI =
+          new ConsentUI(mockInstance, config);
+
+      const placeholder = consentUI.placeholder_;
+      expect(placeholder).to.be.ok;
+      expect(placeholder.hasAttribute('hidden')).to.be.ok;
+
+      consentUI.show();
+
+      // Pop onto the back of the event queue,
+      // so we expect() once our mutate element in show() resolves
+      return mockInstance.mutateElement(() => {}).then(() => {
```

Comment:
Would be better to write this as an async test. Since it is just a test, no need to worry about the code generated to transform async/await into promises. That is:

```javascript
should('...', async () => {
   ...
  await mockInstance.mutateElement(() => {});
  expect(...);
});
```